script.js

```diff
@@ -17,3 +17,193 @@ for (let i = 0; i < mobileItems.length; i += 1) {
     mobileMenu.style.display = 'none';
   });
 }
+
+// projects-recent work
+const workproject = {
+  id: '_1',
+  name: 'Multi-Post Stories',
+  desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has beentheindustry's standard",
+  featuredImg: './images/works.png',
+  technologies: ['html', 'css', 'bootstrap', 'ruby'],
+  liveLink: '#',
+  sourceLink: '#',
+};
```

Comment:
**For desktop version:**

- [ ] kindly increase space to both the left and right sides to match the Figma design.

- [ ] To match the Figma design, you should reduce the gap between the see live and see source buttons and also the name of the project and the technologies.

Your design:
:-------------------------:
![popup_desktop_version2](https://user-images.githubusercontent.com/70483835/208992798-74f364fb-435a-4337-880b-d2ac47501bfe.png)

Figma Design:
:-------------------------:
![popup_desktop_version1](https://user-images.githubusercontent.com/70483835/208992927-e61071c6-368a-420f-a0d7-10cd6533a829.png)

**For mobile version:**

- [ ] Kindly fix the alignment of the "see live" button properly.

Your design:
:-------------------------:
![mobile_version2](https://user-images.githubusercontent.com/70483835/208993698-f999b7ff-981a-4cbb-9afd-3c737612b4d1.png)

Figma Design:
:-------------------------:
![mobile_version1](https://user-images.githubusercontent.com/70483835/208994112-2b93e74a-2f67-4b7f-b6f1-e16d83c0e19a.png)